server/src/models/userModel.ts

```diff
@@ -0,0 +1,53 @@
+import {Schema} from 'mongoose';
+import * as mongoose from 'mongoose';
+
+const userSchema : Schema = new mongoose.Schema({
+    first_name:{ 
+      type:String,
+      required: true
+    },
+    last_name:{ 
+      type:String,
+      required: true
+     },
+    password: {
+        type: String,
+        required: true,
+     },
+    email: {
+        type: String,
+        required: true,
+        unique: true,
+    },
+    location: {
+        type: String,
+    },
+    tech_offer:{
+        type:Array,
+        }, 
+    y_xp:{
+        type:Number,
```

Comment:
I think you said you already changed the model and took that stuff for offer/req out (?), just as reminder I commented

Reply:
I have realised I confused it after the PR, now its adjusted to what we discussed :)